Form-Controls/index.html

```diff
@@ -13,14 +13,66 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <div>
+          <label for="fname">First name</label>
+          <input type="text" name="fname" id="fname" required>
+        </div>
+        <div>
+          <label for="lname">Last name</label>
+          <input type="text" name="lname" id="lname" required>
+        </div><div>
+          <label for="email">Email</label>
+          <input type="email" name="email" id="email" required>
+        </div>
+        <div>
+          <h3>What colour of a t-shirt you want?</h3>
+        <div>
+          <label for="colour">Purple</label>
+          <input type="radio" name="colour" value="purple">
+        </div>
+        <div>
+          <label for="colour">Black</label>
+          <input type="radio" name="colour" value="black">
+        </div>
+        <label for="colour">Green</label>
+          <input type="radio" name="colour" value="green">
+      </div>
+      <div>
+        <h3>Preferable size</h3>
+        <div>
+            <label for="xs">XS</label>
+            <input type="checkbox" name="xs" id="xs">
```

Comment:
I think a user should not be able to select multiple sizes for this product as you have now with `type=checkbox`. Usually I can select only one size. Do you know which input type should be used in this case?